zookeeper/init.sls

```diff
@@ -33,15 +33,8 @@ install-zookeeper:
     - group: root
     
 zookeeper-home-link:
-  alternatives.install:
-    - link: {{ zk.alt_home }}
-    - path: {{ zk.real_home }}
-    - onlyif: test -d {{ zk.real_home }} && test ! -L {{ zk.alt_home }}
-    - priority: 30
-    - require:
-      - archive: install-zookeeper
   file.symlink:
     - name: {{ zk.alt_home }}
     - target: {{ zk.real_home }}
```

Comment:
Yeah, you're totally right. That's what I was talking about: there was an attempt to fix Salt issue with the workaround in the formula.